scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -418,15 +418,17 @@ trait ClassLikeSupport:
       memberInfo.get(name).fold(argument.rhs.dokkaType.asSignature)(_.dokkaType.asSignature)
     )
 
-  def parseTypeDef(typeDef: TypeDef): Member =
+  def parseTypeDef(c: ClassDef, typeDef: TypeDef): Member =
     def isTreeAbstract(typ: Tree): Boolean = typ match {
       case TypeBoundsTree(_, _) => true
       case LambdaTypeTree(params, body) => isTreeAbstract(body)
       case _ => false
     }
 
+    val memberInfo = unwrapMemberInfo(c, typeDef.symbol)
```

Comment:
Why do we need `memberInfo`? It does not seems to be used anywhere in code.

Reply:
My bad, fixed